### PR TITLE
[updated with honolulu fixes in master] Translation Checkins

### PR DESCRIPTION
**Pull request by vetrivelc**
L10N:CP-23040,CP-23012,CA-255548 & translation updates

### CODE REVIEW

File: XenModel/Messages.ja.resx [hunk 1]
```diff
@@ -2279,7 +2279,7 @@ SR の物理使用量が {2} を超えるとアラートが送信されます。
     <value>(管理インターフェイス '{0}' により使用されているため、このネットワークの MTU を変更できません)</value>
   </data>
   <data name="CANNOT_CONFIGURE_JUMBO_VM_NO_TOOLS" xml:space="preserve">
-    <value>(XenServer Tools がインストールされていないアクティブな VM が接続されているため、このネットワークの MTU を変更できません)</value>
+    <value>(XenServer Tools がインストールされていないアクティブな仮想マシンが接続されているため、このネットワークの MTU を変更できません)</value>
```
Comment (MihaelaStoica) [inline XenModel/Messages.ja.resx]: Although unrelated to this change, I have noticed the text "XenServer Tools". This should be "[XenServer product] Tools" to allow rebranding

Comment (vetrivelc) [inline XenModel/Messages.ja.resx]: @MihaelaStoica Fixed.

Comment (vetrivelc) [inline XenModel/Messages.ja.resx]: Commit # 5abd456

Comment (MihaelaStoica) [inline XenModel/Messages.ja.resx]: @vetrivelc Commit # 5abd456 doesn't contain any update to this file

**Pull request merged**